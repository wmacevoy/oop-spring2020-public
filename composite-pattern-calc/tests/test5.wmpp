1
((((((d*(v+i))+j)+(((r*610)+(((((931*491)*(v+p))*((948+9)*p))*(((6+y)*(3*106))*8))*i))*((((2*((j*v)+(n*2)))+(((z+0)*1)+n))*(((c*(r+219))+((s*3)*3))*(c*((7+0)*(512*3)))))*(((((q+j)+(489*m))*((t+5)*(v*t)))+(((4+l)+(f*k))*((1*z)+e)))*c))))+(((c*(f*399))*((((((u+r)*(8+z))+((w*8)+(416+920)))+(((t+6)+(x*375))*((5*169)+(a*9))))*b)*((((651+4)+(z*(3+9)))+((4*(y*0))*5))+((((n*5)*(5+7))+((y+3)+(337+b)))*(((3+c)*(956+9))+((0+v)+(h+2)))))))+5))+(8+((((m+v)*(((((f*1)+(8+9))*((z*l)+(2+g)))+0)+(4*(((9*k)*(v*r))*7))))+((x+((((20*b)+(1+433))*((340+n)+(4*h)))*(e*((7+6)+(4+1)))))*(((((c+v)+(k*x))*(5+(l*w)))+(x*(1*v)))*(((2*l)*((8*8)+(d+4)))+l))))*(((w+t)*(((((1*5)+9)+((g+m)+(y+x)))*(((p*965)*(c+6))+274))*((((e+t)+(v+3))+((c+o)+(y*i)))+(((r*k)*(910*r))*((1*i)*(m*200))))))+(4*(((((w*a)*(6+9))+o)+(((t*6)*(4+e))*((i*2)+(642*b))))+(r*5)))))))+(((8*(((d+((2*((16+6)+6))+(((3+g)+(n*0))+((7*6)+6))))+173)*((7+((((2*8)*234)*((s*g)*(s+4)))*(((599*7)+(9+6))*((656*3)+(p*122)))))+724)))+((((9*(861+3))+4)*((3+((((9*y)+(l*s))+((248*x)*(x*b)))*((x*(z+w))+((5*k)+q))))+6))+(b*((((((8+0)*(a*m))*(g+(f+f)))*(((4+e)+(q+7))+((0+960)+(d*x))))*t)+(((4+1)*(((o+5)*(4*5))+e))*2)))))+(((((0+((((4*4)*(a+6))+670)+e))*(((((x+0)+(o+w))*((7*359)*q))+(((2*3)*(p+9))+((f+210)+(e*y))))+((a*((d*8)*z))*8)))+1)+c)+5)))
(((((6*s)*((((((574*v)*((p+z)*i))*(((b*s)*(4+u))*((y*j)+(7+d))))+m)+(((((1+h)*(5*y))+948)*(((777+273)*(2*9))*((6+q)*(9+y))))+((((7+u)+(1+l))+((e*x)*(568*93)))+(((k+b)+(3*a))*((589+7)*(1+j))))))+((((((j*4)+(p*n))*((e*o)+(2*4)))*((x*(z+9))*((f*c)*(r*8))))*((((i+c)+(8+2))*((9*y)+g))+(s*((2+b)*(z*8)))))*((((p+(7+1))+b)*h)*((((l+l)*z)*((z+a)+(m+270)))*v)))))+((((((((8*793)*(o*y))+((k*9)+799))*a)+(n*(((2+5)+(i*b))*((7*e)+(1*1)))))+(((7+((3+3)*(d*n)))+(((3+r)*(2+1))*(a*(x*0))))+(i*((9+798)*((1*7)*(h+9))))))+(((((7*r)*((c*c)+(251*s)))*(((0*5)*(2*421))+((8+3)+(a+9))))*((((j*l)+(u+4))*((3+m)*4))+(((l+1)*(374*2))*((572+d)+(8+t)))))+((8+(((5+5)+549)*((l+m)+m)))*5)))*(((p+((((x+h)*(v*i))+((d+936)*(9*3)))+(((302+y)+6)+((5*737)*1))))+(((((l+p)*(v+g))*((g*m)*(0*945)))*(((421+2)+(1+r))+((3*7)*(h+p))))+(((265+(766+9))*((941*9)+(t*e)))+(((307+9)+(n+n))*((4+y)+(d+3))))))+(9*(w+((((l*3)*(4+2))*((4*4)+(z+5)))*(((n*k)+(p*1))+((467+2)*9))))))))+(((((((((6+7)+(x*6))*((973+4)*(n+7)))*(v*((9*w)+(k+135))))*(((o+(s*0))+((165*m)*1))+(((b*s)+(j+u))*((8+d)*(3+4)))))+(((((693+1)*(r*6))+c)+(((i+e)+(w+77))+((y+6)+(z*9))))*((((6*g)+(8+f))*((g+e)*(7+l)))+(l+((m*z)*4)))))*((((((q+l)*(r*y))*j)*(((u*4)+(4*4))*f))+((((20+7)+(e*1))+((i*0)+(e+21)))+(((e+y)+(6+f))*((q+6)+(y+4)))))+(((((3*475)+(2*y))*((358+r)*(8*x)))*(((3+6)+4)*(149+(522*c))))*((((6*s)+(b*2))*((m+0)*(c+5)))*(a+((i+s)*(v*q)))))))*(s*3))*((2*7)*c)))*105)
(((((((9+((((6*r)*(1*v))+5)*(((d+0)*(7*1))*d)))+6)+((((1+((5+j)*2))+((3*(102+i))*1))*((x+r)*(((0+8)+(p*j))*((0*y)+(0+r)))))+(((((b+n)+(h*8))+((155*h)+(8*m)))*((k+(t+5))+((9*z)+(h+552))))*((((z+u)+(d*414))*4)+(((9+p)+(g*h))*((h+744)*746))))))+(((((((596*7)*(6*5))*((s*3)*(4*f)))+(w+((t+c)+(e+9))))*(3+(((435+214)*(3+9))+((9+139)+l))))*(((((j+7)*8)+(165*(7*q)))*(((p+s)+(a*3))+((3*3)+(8*s))))+(h+((9*(w+b))*((t+5)*(k*f))))))+(6+130)))+((m*((s+((((w*9)+(j+l))+((w*z)*(237*v)))*(((7+b)*(g*s))*((419+669)*(49+3)))))+((((p+(6+k))*((i*0)*(0+x)))+(((992*374)+(831+o))*g))+(659+(n*z)))))*(x*(((((l*(9+t))*(5+(o*b)))*(((122+t)+(m+p))*((3+i)+(6+t))))*((p*((a*n)*c))+(9*(y*(6+4)))))+((8*(5+((0*766)*(h+x))))+(c*(((y*o)*(m*b))*4)))))))*(5+e))+((((((4*((((7*8)+z)*((2*413)*(5+m)))*(0+((z+359)*(0+a)))))+((d*((b*(495+1))*((5+y)+(h+t))))+((((f*418)+6)+6)+(1+((j*3)+(8*k))))))*(((182+((9*(j*0))+((271*n)+(p+g))))*((((j+9)+(5+q))+((u+179)*(7+k)))*9))+(546+((((0+394)*(y*h))*(w+(1+l)))+(7*((n*p)*(n+x)))))))+(((((((451+n)+(6*u))+(j+(0*8)))*(3+((u*h)*(m+s))))+((((8*8)*(2+q))+9)*(((n*m)*(r+5))+(a*(99+468)))))*(j+((((6+0)+(e+x))*0)*((a*(4+p))+((419*2)+l)))))*((y*((((g*410)+(h+4))+((7*8)+(x+50)))+(((265+4)+(991*0))+((308+0)*(h*7)))))*(((((s*0)*a)+((g+k)+(s+5)))*(((v+w)*(v+b))*((c*j)+b)))+((2*976)+(b*((197*p)*0)))))))+(((((((1+(7*q))+((a*w)*(9*5)))*((6*(8*4))+((t*z)*7)))*(f+(((5+u)*58)+l)))+(2*((7+(v*(e+f)))*(u+(s+(r+q))))))+(((j*(((6+i)+(u*n))*(q+(i+480))))+(r*(n+(e*(8+t)))))*((5*((j*(k+l))*((x*774)*(y+7))))*((((u+d)*(200+2))+((l+70)+(k+n)))*(v*((580*1)*(7*t)))))))*8))+(i*(5+((((((d+a)+((9+860)+(298+5)))+((537+(w*2))+((5*972)*(y*h))))*(332*468))*(((((7*s)*(9+8))*(4*(5+3)))+(((2*802)*(3*s))+((9+d)*796)))+((((318*3)+(0+877))*((100+4)*(200+2)))*(j*((5+v)+(4+r))))))*(((((h*j)*((i+s)+(9+d)))+(((3*g)+(d*7))+((y*363)*(3*k))))*(b*i))*((298*b)*((((445+3)+(9+f))*(4*(9*8)))+(7*(2*(4+541)))))))))))
((d+((((921*(1+(671*(((m+u)*(m*4))+((b*d)*(9*0))))))+((((((o*1)+y)*(e*9))+(4+((486+l)+(9*t))))*u)*(((((t*f)*(2*e))*((2*l)*a))*(w+((92*f)*(h*318))))*((((7*z)+(k*7))+((g*l)*(650+6)))+(((o*y)+(4+c))+((8*g)+j))))))*((3+((8*(((3+0)+(6+z))*(i*(t+822))))*((((x+5)*(6+3))+((h*8)+b))+(((1*h)*(c+s))+(593*(547*51))))))+((((((r*q)*(e*h))*(v+(y*3)))+(((2*1)+0)*c))*(t+((p+(7+z))*(7*544))))+((p*z)+3))))*((5*410)+((((4+b)*(((s+h)+(t*b))+8))+(i+(e*(((1*n)+(a+v))+((e+246)+3)))))*479))))+((0+((((((((v+273)*(w*9))+4)+787)*(z*(((4+c)*(8*y))*116)))+((((883+(6*6))*((e+8)+(l*r)))*(((n*j)+(s+2))+((3*f)+(s+3))))+((((t+6)*(f*3))*((2+v)+g))+(((q*969)*(p*3))+((745*7)*(j+390))))))*((((((638*7)+(k+h))*(p+w))*(((7+l)*(3+z))+6))*(1+(5+((q*s)+(400*t)))))+(((((i*s)+(28+4))+((m+6)+(9+7)))+(((4+p)*(0*304))*((2+365)*(a*6))))*(((4+(u*e))+((v+9)+y))*(((5*h)*(s*5))+((6*121)+(5+2)))))))*(4*2)))*(((((((883*((8*m)+(244+a)))*(((j*8)*(30*n))*((u+s)+(6*633))))*((((k*3)+w)+j)+((p*(y*1))*((177+506)*(q+p)))))+((y+(((7*v)*(7*u))*((g+0)*(z+7))))+(r*4)))*(((8+(((5+w)*2)+(h+e)))*((((h*x)+(d*5))*((2+c)+(y+d)))*(((4*o)+(r*d))*((j*b)*(s*z)))))*(2+((1*((6*d)*a))*(k+((8*a)+2))))))+5)+(((((6+(o*((x*a)*(7+g))))+((455*(1+(1*x)))*(9*(3*(1+r)))))+0)*((((((1*8)*(9+h))*((m*h)+4))*(q+((c*r)*(w+4))))*(z+(e*((6*h)*(316+h)))))*(((((6*6)*(317*9))+((l*6)*(418+o)))+(((7*z)*5)+((1*2)+(v*874))))*(o*(((1*i)*(592*7))+((s+838)+(w*s)))))))*(((((((u+3)*7)+((q*1)*(n+1)))*(((s+h)+1)+(o+(t+a))))+w)*(((((8*9)+(9+u))*((z+3)+(8+l)))+(((0*h)+d)+((a+523)*(3+t))))*((((b*x)*(6+u))+((s*j)*(0+s)))*((i+(u+g))+((4*0)+(1*h))))))*((((((6+o)+(8*m))*((2*k)+x))*(((706+5)+(c+l))+((f+9)*(0+9))))+((((x*y)+r)+6)*(((8+0)+(b+2))+(c*(4*4)))))+(((((4+k)*(6+9))*((g*728)*(x*r)))*((8+(x*1))*((h+481)+(1*9))))*((((y*8)+(9*s))*((7*4)*(s+l)))*(7*(6*(574*i)))))))))))
3
(279*((((((((((l+m)*382)+6)*(b+((g+i)*(6+t))))+((((479+7)*(l*9))+9)*(875*((r*1)*(r*1)))))*(((((9*e)+o)*((7*6)*(u*n)))*(((k+186)*(w*8))*((e+8)*3)))*((((l*0)+(l+n))*((8*d)+(146+i)))*((p+(d*0))+((k*k)+(q+m))))))+9)+(x*(o+(((r*((6+47)+(7*2)))*(((7*b)*(v*7))*0))+((((2*h)+(317+q))+((a*0)+v))*(1+((v*t)*(o*b))))))))+(((((551*t)+u)*((((2+(1+142))*((1+6)*4))*(((n*5)+(9*u))*((k+2)*(o+3))))+(o+7)))+((((((r*0)*j)*(8+(2*n)))*(((263+u)*(f*y))*((363+i)*(7+h))))*8)+(r+((((g*283)*(s*5))+((1*t)+j))+(n*((2*o)*(3*o)))))))*((913+(((7+((623*b)+(r*4)))+e)*((657*7)*(((4+o)*(6+0))*6))))*((g*((t*((2*g)+(9+7)))*(((b*r)*m)*((5*b)*7))))+((4*(((4+3)*(z*a))+((p*8)*(k*202))))*1)))))*((q*((((((762+(k*w))*v)*(2*((n+i)+(f*j))))+((((3*j)*(7+9))*n)*(176*((4*255)+(5+u)))))+(k+((((6*6)+(8*9))*((f*5)*(451+w)))+(((642+7)+(2+p))*((325+2)*(9*c))))))*(957+(((0+((331+457)*(0+h)))+((q*(4+q))*((l*3)*(1*u))))+((((6*m)*(w+m))+(b*(771*n)))+(o*((v+8)*(6+4))))))))+((((((2+((0+q)+(2*q)))+(562+((m*2)*(e+n))))+c)+(x*(h+(((g+0)+(7+b))+((p*u)+(8+h))))))+1)*(((((((3+5)+(h*9))+((6+d)*(6*9)))*(((g*9)+(w+8))*((j*y)+(0*0))))*y)*(5+((((i+1)+(2*5))*((5*g)+5))+7)))*((5*717)*(((861*((5*n)+(q+9)))+(((4+6)*(p*o))*g))+((((192*7)+(1*j))+((3+3)*(n+w)))+((o*(4+r))+1)))))))))
k
(((((9+(((p*(3*311))+((((x+k)*(3*3))*((9*u)+(o*e)))+(((204*o)+(b*3))+((0*p)*(8+3)))))*(n+((((5+5)*(q*7))+((5*l)+713))+(((j*u)+(m+1))*((e*1)+(7+h)))))))*((3*(465*((f*((w*8)+(674*y)))*(((k+8)+(y+6))*((326+9)*8)))))*((((g*((s+i)*(q+0)))+(((1+t)*(787*1))*((2*m)*(p+c))))+((((5*k)*(x+3))+((w*7)+(5*s)))*(((264*y)+(5+y))+((k*p)*(a*0)))))+2)))+(((((((x+(v*b))*((0*9)+3))*(((682*2)*(2*m))*((4*1)+(d*j))))+((((261+2)+(0*621))+((1*h)*(k+n)))+(7*((w+w)*(3*7)))))+f)*(0+c))*((q*((5+(265+(5*(7*8))))+((((e*b)*475)+((659+v)*(8*s)))+(((l*0)+q)*((c+0)*(492+t))))))*((((((2+8)*(798*g))+((7+4)*(l+4)))*((5+(0*g))*((4*s)+781)))*(((351+(9+h))*(l+(7+5)))+(c*((0*6)*(8*0)))))+(((6*(i*(j+y)))+c)*((2*((k+1)*(560*g)))*((t*(s*j))*((e*3)+5))))))))+((m+((480*(w*((((i+t)+(947*c))*((e*y)+(744+n)))+(((h+804)+(n*960))*((1+h)+(7*0))))))+((((((x*5)*(i*2))+d)+(w*3))+((((3*r)+(d+0))+y)+((a*(661+x))*((3*0)*(v*t)))))+(2*(((6+(k+j))+((9+4)+(307+z)))*(l*((3+4)+(6*h))))))))+((((((((e*n)+(l+m))+(s*6))+(((5+1)*(w+9))+((r+t)*(d+z))))+((((8*5)*g)+((p+725)*(905+6)))*(((z*4)*(z*1))*((i*i)*(346*y)))))*w)+(((((117+(p*d))*2)+(5*((f+1)+(9+938))))*((((4*e)+(5+9))*((8+g)*(i+u)))*((x*(n*4))+((4*9)*3))))*5))+p)))*u)
((d+5)+m)
((((((488+h)*((((((8+182)*(m+f))*((4+v)+(s*p)))*(((6+t)+(101*457))+c))*((((i+4)+(y+y))*430)*(((d+g)*(h*795))*((s*7)*(x*779)))))*((((n*(g+3))+((m+t)*(n+492)))*(l*((7*d)+(g+j))))+(9+((f*(8+w))+((c+7)*(y*l)))))))+5)+((((((((p*t)*i)+((a*695)*6))+(((v*269)*(8+l))+j))+((s+6)*(((2+v)+(2*5))+((b*e)*(80*t)))))+(4+((((l+4)*(8*t))*((w+2)+(q*s)))+7)))*((3+((o+6)+5))*(((((3*5)*(i+818))*288)*(1*((k+778)+(3*5))))*426)))*((5+d)+(c*(((((j+9)*(8+942))+8)*(((9+9)+(830*m))*((z*5)*(7*7))))*(((r+(912+7))+(4*(465+d)))+6))))))+((((((c*((7*(o+d))+l))+((o+w)*(((s*3)*(0+0))+((1+i)+(w*h)))))+(((((6*n)*(7*7))*((e+9)*(y*2)))*(974+((l*x)+(2+b))))+((((5*h)*(p+1))+((c*6)+(j+7)))+(((m+s)*q)+((w*0)*d)))))+(1*(((((1*d)*(x*a))+h)+(7+((6+7)*(0+909))))*(922*(a+((h+p)+(4*j)))))))*((((3*(1*((z*a)*(1+k))))*110)+(((((r+4)*(0*q))+((5+800)+(m*e)))+(m*((j+f)+(7+t))))+411))+(e+516)))+((i*((((g+((t+5)+(w*1)))*(((n*m)+0)*l))*((2+((6*3)*(9+9)))+(((z*n)*(c+g))*8)))+2))+(((((((d*d)*(0*p))+((t*3)+263))*(((x*1)*(e*w))+(d*(w+y))))+((((t+u)*(0*371))+((0+6)+(i+n)))*(3+((8+f)+(7*h)))))*(((((a*l)*(9*1))*((d+w)*(f*d)))+(((3*j)+t)+(g*3)))+((4+((a*d)*(0+678)))+(0*a))))*1))))+((((((27+n)*((((4+(529*5))*((0+u)+c))*(((2+q)*(f+y))+((h*e)+(p*v))))*g))+((((6+((5*5)*g))+(((8+m)*1)+((s+a)*(o+722))))*((3*(4+(782+5)))+(((q*x)+2)*880)))*5))*(223+(((((u*(y*a))+((472*p)*(u*o)))*((r+(831+x))*((u*9)*(i*569))))*((8*((f*q)*(223+h)))*((581+(s*d))*((y*o)+(8*a)))))*(y*(f+5)))))*((((((((p+o)*(8+5))*((l+416)+(f+7)))*(((2*1)+(k*870))+((4*z)*(c+0))))+(1*(((g+1)+(6*n))+(2+t))))*(((915+((117+0)+(2+0)))*(((i*0)*(5+r))*((784*x)*u)))+((7+((k*v)*(q+x)))+x)))*((d+((((t*g)+c)*((b*l)*(951*8)))+(((j*d)*(6+t))+((588+8)*(i*5)))))*w))+q))+((7*(0*(((n*(((439*f)+(t+a))+((613+3)+(3+d))))*(s+(((u+o)+(j+3))*((6*425)*601))))*(((((6+g)+(8*l))+639)*(((z*l)+(181*i))+((370*4)+(8+r))))+((((z+5)*(s*w))*((2*p)*(u+80)))*((2+(u*f))*((h+8)*(c+0))))))))+(((((9+825)*((((t+8)+4)+((4+906)+(73+f)))+(425*((5+8)+(385+6)))))+(((((2+1)+(u*e))*((j+799)*(321+5)))*(m*((5+r)*(0+4))))+(o+(0+((h*j)+(6*9))))))*((2+(5+(((1*2)*(6*390))+((7+3)*(m+i)))))*(((0+(4*(z+x)))*(((b+4)+(5*41))+((c+290)+(p*k))))*(((4*(3+772))*((5*4)*(0+h)))*((o+(a+692))+(969+s))))))+(((((m+((s+w)*(486*0)))*(v*1))+949)*e)*946)))))
(3+(((((((3*(9+((8+154)+(6*y))))*((((f*9)*(273+w))+((2+8)+(t*p)))+(((2*9)*(2*4))*(697*(4+n)))))*((y+(9*((901+7)+(4*o))))*(v+(((8+m)+(f*7))+((0+h)*(u*n))))))*((m+(q*(((r+0)*(293*4))*9)))+(((((c+u)*4)+(y*(7*9)))+(s+294))+((((7*m)*(6*5))*((c*j)+(w*4)))*(y*((w+e)+(c+8)))))))+397)*(((360+(w+6))+(((((3+(g*721))*((m*0)*w))*((u*(m*x))+((e*x)+(p*8))))+((((195+g)+d)+((850*2)*(n*1)))+(((e+m)*(1*p))*(9+(a*912)))))+(((((3*t)*(o*l))+((0*6)*(343+u)))*((111+(t+m))*9))*((((y*n)+(8+9))*((a+e)*(j+q)))+0))))+((((((1*(23*h))*((2*3)+(3+3)))+(((273+m)*(667*503))+(w*(9*b))))*((((7*3)*(3*398))+((o*g)+(p+7)))+((n+(4+0))*((9+a)+(546+6)))))+(((((3*n)*(5+t))*((c*a)+(n*g)))+(((6+n)*(i*5))*((i+y)*(a+7))))*((((d*o)*(f*2))+((0*z)+(s+c)))*(((7+6)+(9+s))*434))))+((((((0+4)+(2*386))+((l*t)+l))*(((7*521)*w)*((j+716)+(c*x))))*((i*((v+u)*(3*1)))*(((y+z)+(j+r))*7)))*((3*(((h+o)+t)*((1*n)+(9*3))))+((((c*e)+(9*551))*4)*p))))))+(((2+(k*(((((v*8)*(5+g))*(f*(5*y)))+(((6+7)*(4+2))+((f*6)+(8*t))))+((((p+9)+(h+t))*((0*e)+(3+5)))*k))))*(((((8*((5*b)+(i*i)))*(((y+q)+(6*9))+(r*(c+3))))+((((g+q)+(8+p))*((x*g)+e))*(((1+2)+(0*m))*((2+q)+(3*8)))))*(((((9+653)*(g*v))+h)*197)+441))*h))+((((((((u+l)*(9*p))+((256+996)*(6*457)))*o)*(((1+(1+t))+m)+(((r*58)*(j+626))+g)))*((8*(((223*472)*3)+((3+6)*(o+2))))*((((s*2)+w)*((8+q)*(937*d)))*b)))*((((((637+q)*(8+c))+((5*z)*8))+(((k+9)*(n+f))+3))+(((h+(200+l))+((7+2)*(9+n)))*(((w+4)+(71+r))+(937+(e*9)))))*(((8+((k+v)*(201*w)))+h)*(3+(((6*u)+(0+5))+((q+h)+h))))))+v))))
(((y+z)*(((((((((774+b)+(1+1))*((z*o)*(7+5)))+e)*6)+((4+(((w*532)+(9+s))*(7+(t*4))))*((((w*c)*(7+444))+765)+7)))+((((((o*a)+(c*d))*((k*m)*(w+u)))*(5+(k+(4*3))))*((z*((n+x)*(o+5)))*(((s+w)+(2+b))+((7+j)+(j*h)))))*((((d*(k+e))*((p+9)*o))+(w*((6+d)+(4+e))))+6)))+(((((((8*0)+6)+((6+c)*(736+7)))+(((k+9)*(5*7))+((o*8)*(j+7))))+((r+((h+607)+p))+((7+(5+p))+((i*k)*(850*117)))))+(((((s+v)*123)+((0+4)*(7+a)))*(6+((x*s)*x)))*((((u*l)*b)*((0+u)+d))+(((368+d)*(152+s))+((n*s)+(l*o))))))*(((((460*(8*8))+((l*h)*(s*t)))+(500*((x+m)*n)))*(5+(((u+9)+(850+9))*((n*0)+(p*979)))))*(m+((((9+m)*(8*754))*2)+(((z+l)*(632+6))*((z*5)+(3*9))))))))*8))*(((s+(5+((114*(((f+(u+1))+(6*(s*z)))+(3+(w*(d*591)))))*((557*1)*((((9+c)*(h+o))*((q*d)*(t*v)))*1)))))*(((r+w)*((((((d+8)+(6+v))+(l*(o+363)))+(((l+r)*(t+0))+((y*i)+(595+6))))+q)*(((((a*w)+(375+t))+f)+5)+((((4*m)+(y*9))*((9*z)*0))*6))))+726))*(((((((((5*k)*u)*((207+9)+(m*8)))*(((o+o)+t)+5))*p)+(((((496*z)*(k+r))+4)*(((115+r)*(e*e))+((j+r)+b)))+((((1*t)*a)+((t+x)*(y+9)))*6)))+(((((x*(w*135))*((f+2)*(3*777)))*j)*0)+(((((c+3)+(2*x))*((g+866)*(1+v)))+(((d*6)*(5*6))+554))+((((f+1)+(2+x))*7)*((u*j)*((d+p)*(2+n)))))))*(f+(((5+(((u*8)+(6*b))*((8+9)*t)))+(5*u))+(((((o+882)+(y*389))*((791*p)*(h*1)))+(((c+e)+(j*3))*d))+5))))+(((((863+(b*((b*0)+(b*o))))*((((9+j)+(0*5))+((f+595)+b))+(((c*y)+(6+9))*((g*e)*(t*7)))))*438)+297)*(((((a*((n*0)*(299+4)))*(((8*2)+(n*v))+((8+2)+(d*l))))+((t+((218+b)*(766+7)))+((6*(5*f))*((d*p)+(7*t)))))*(719+((((u+8)*(558*t))*((1*l)+27))*332)))*((((m+((b*f)+3))+(j*((q+4)*9)))+(s+f))*(((((y*4)*4)+((4+4)*(l*h)))*(((r*9)*h)+(2+(p+p))))*((p*d)*(7+((t*x)+(m+385)))))))))))
(((a+((((((((1*422)+(1+853))+((4*b)+(2*u)))+(((296+o)+(a+2))+w))+((8*(r+(k+w)))+b))*(((9*2)*(((1*p)+(3+824))*((821+5)*(9+6))))*(x+((6+(724+k))+((d*1)+(u+723))))))+(t+(t+(0*(((a+4)*(o*w))*(m*(1*r)))))))+(((((((j*1)+r)*((w+p)*(h+3)))+e)*((((h+8)*(p*114))*((t+761)+(743+d)))+o))*((6+(b+((u*e)*(1+g))))*((((d+g)*(b*5))*g)+(((x+1)*(9+9))*(v+(p*n))))))+((315*((((8+5)+(6*345))*(0+(2+291)))+(((e*z)*(q*3))*((d*6)+5))))*(((((v*q)*g)*((4*6)*(7*628)))+(w+((i*d)*(m*d))))+k)))))+(d*2))*y)
((q+((136*0)+((((r*((((p*r)+(3+k))+((0+5)*(9*954)))+(((368*l)*(324+u))*((7*7)+(r*k)))))*((((811*(u+1))+((z+t)+(a*a)))*(((2*k)+(d*144))+((q*3)+1)))*2))*((((((d*4)+(v+2))*5)*(((e+190)*(u+k))*(o*t)))*p)*(((((w+547)+(s+z))*((u*400)*(v+989)))*(((q+j)+3)*(2*(k*9))))+((((y*b)*(4+a))*(n*(6+q)))*(((w*p)*(8*8))*((1+2)*p))))))+1)))*((((((((((9*5)+(f*2))+((z+8)+(0*k)))*((g+508)*(783+4)))+((((q+l)+(p*k))+((7+r)+(f+6)))*(((d*q)+(e*r))*((784*139)*(u+6)))))+((((z*(w+g))+7)*(y+7))*((((9+1)+(q*q))+(m*9))*(((5*4)*(226*m))*((5+3)*(z*l))))))+b)+(a*(6*(((((3+p)*(2+1))+q)*(0*(0+(2+4))))*(336*(((5+2)+(k*55))+((9*638)*(t+3))))))))+((((((((517*8)*(802+6))+((7+166)+(i*c)))*(((544+r)*(t+f))+p))+((((4+1)+(2*4))*(0*(l+z)))+(((c*k)+792)*((e+r)+(i+g)))))+(((8+0)+(((k*879)*(8+5))*((2*j)+(z+653))))*867))*((((((7+72)+(j+542))*c)+(((z+557)*(o+7))*((5*k)*(u*277))))+((((905*w)*(y+8))+((s*q)+u))*(((7+h)*(433+2))*((5+e)+(2*8)))))*((((2*(m*b))+((4*u)+e))+(e+((r*9)*(m+i))))+((((r+2)*(5+l))*2)+(((755+696)*(d*q))*8)))))*(((327*((((f+r)*(9+248))+((936+r)*t))*(((3+579)+(n*j))*a)))*6)+((((((9+c)+(3*7))*((p*4)+(l+t)))+(((q+9)+(i+304))+((0*9)+(5+8))))*((((452+k)+(3+5))*t)+m))*((q+(((203*8)*(s*i))*q))+8)))))+(((a*((((((o*r)+(x+y))+((q+0)+(7*o)))*6)*((((c+l)*(y+749))+(c*(k+l)))+(((621*w)+(196*z))*((1+m)+(w*7)))))+((8+(((d*605)*(9+8))+((e+2)*(5*x))))+5)))+((s*(((((3+y)+x)*721)*(((5*s)*(f+b))*((w*x)*(8+6))))*((((6+f)+(o*k))*((3*v)+705))*(((5+2)*z)*((n+k)*i)))))*376))+(((52+(6*((b+((0*5)*(4*q)))+(((n+7)+(w+7))+((945*z)+446)))))+((((((l*2)*(7+w))*((489*u)+y))*5)+((2+(706+7))+(((7*5)+(z+x))*((s*0)+3))))*(((((s+572)+(q+l))+s)*(((f+p)+(1*533))+((f+a)+(v*0))))+((w+((111*u)+(6+600)))*(((6*1)+(o*y))+((5+5)*(5*9)))))))+(((((((8*9)*(q*6))*((6*174)+5))+(((m*0)*(2*7))+((a*1)*(x*p))))*((3*q)*(((y+r)*(f*2))+((2+l)*(h+501)))))*(((((9*g)*(z*0))+(h+c))*l)*(((v*(s+s))*(1*(2*v)))*(w+7))))*((((((1+t)+(a+v))*((8*s)*(8+n)))+(((2*b)+168)*((7*g)*(4*2))))+(v+9))+(q*((((w+5)+(h+a))+8)*(((v+i)+w)+((1*5)*(s+651)))))))))))
