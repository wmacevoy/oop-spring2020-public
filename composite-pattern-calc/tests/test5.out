1.0
2.35256381E10
1.10702782E15
7.008526E13
1.043065E26
3.0
4.1848866E22
0.009345794
9.8449235E8
5.0191746
3.007806E22
1.0092363E28
Infinity
3.48686202E12
9.866991E28
